extensions/objects/templates/literally-canvas-diagram-editor.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the literally canvas diagram editor.
+ */
+
+describe('LiterallyCanvasDiagramEditor', function() {
+  var LCDiagramEditorCtrl = null;
+  var mockLiterallyCanvas = {
+    setImageSize: function(width, height) {
+      var text = (
+        'The updated diagram width is ' + width +
+        ' and height is ' + height);
+      return text;
+    }
+  };
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.inject(function($componentController) {
+    LCDiagramEditorCtrl = $componentController('literallyCanvasDiagramEditor');
+    var mockDocument = document.createElement('div');
+    mockDocument.setAttribute('id', LCDiagramEditorCtrl.lcID);
+    document.getElementById = jasmine.createSpy(
```

Comment:
You should not be redefining "system" functions like document.getElementById. Please work with @marianazangrossi to get a better solution.

Shouldn't you be using angular.element btw instead of document.getElementById? In general AngularJS doesn't encourage direct access of the DOM.

Reply:
I have replaced the test so that the system function is not redefined. But as for using angular.element instead of document.getElementById, it doesn't seem to work as expected. The error is ```Uncaught TypeError: el.removeChild is not a function```. I think Literally Canvas uses remove child and the angular element doesn't have removeChild as a method.